cephfs/path.go

```diff
@@ -44,3 +44,68 @@ func (mount *MountInfo) RemoveDir(path string) error {
 	ret := C.ceph_rmdir(mount.mount, cPath)
 	return getError(ret)
 }
+
+// Link creates a new link to an existing file
+//
+// Implements:
+// int ceph_link (struct ceph_mount_info *cmount, const char *existing,
```

Comment:
This should be indented an additional space. This will cause it to be formatted as a block in godoc. This applies to the other new functions as well.